other/remove-hostpath-volumes/remove-hostpath-volumes.yaml

```diff
@@ -48,7 +48,7 @@ spec:
                 op: remove
           - list: request.object.spec.containers[]
             foreach:
-            - list: element.volumeMounts
+            - list: " element.volumeMounts || `[]` "
```

Comment:
Good catch on this, and good to see a test for it.

Reply:
Thanks!